doc/api/net.md

```diff
@@ -512,14 +512,17 @@ If `true` - [`socket.connect(options[, connectListener])`][`socket.connect(optio
 haven't yet finished. Will be set to `false` before emitting `connect` event
 and/or calling [`socket.connect(options[, connectListener])`][`socket.connect(options, connectListener)`]'s callback.
 
-### socket.destroy()
+### socket.destroy([exception])
```

Comment:
Two bits:

1. What is the expected type of the argument
2. Either `reason` or `error` would be better than `exception`